web/package.json

```diff
@@ -15,11 +15,15 @@
     "async": "^2.6.0",
     "bootstrap": "^3.3.5",
     "font-awesome": "^4.5.0",
+    "less": "^3.0.1",
```

Comment:
less and pug are dev and not runtime dependencies

Reply:
done